lib/jamaica/games/mad_sentence_game.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:animated_text_kit/animated_text_kit.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class MadSentenceGame extends StatefulWidget {
+  final List<List<String>> words;
+  final List<String> header;
```

Comment:
all games should take onGameOver parameter

Reply:
onGameOver added